_data/errors.yml

```diff
@@ -125,7 +125,7 @@ oidc:
       accordion-id: token_expired
       content: |
         ##### Why it's happening
-        In the authentication response, a `code` value is returned that is then used in the `token` endpoint request. "The code is intended to be used within 15 minutes immediately after it is generated. This error may mean that the user paused for too long during the authentication attempt.
+        In the authentication response, a `code` value is returned that is then used in the `token` endpoint request. The code is intended to be used within 15 minutes immediately after it is generated. This error may mean that the user paused for too long during the authentication attempt.
```

Comment:
I know this wasn't part of your changes but while we are here I would say 
`The code is intended to be used within 15 minutes after it is generated`
To reduce chances of confusion but I defer to @karenstrother  on language

Reply:
I think the language could be stronger too. It's more than an intention, the code _will_ expire.

Reply:
updated in https://github.com/18F/identity-dev-docs/pull/398/commits/4bb0d783af1e2c2a834fc8fcaadfe47a09c24370

Reply:
This is on me - I suggested the change to "The code is intended to be used within 15 minutes immediately after it is generated."